spec/routes/redirections_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+require "decidim/dev/test/spec_helper"
+require "decidim/core/test/factories"
+require "decidim/proposals/test/factories"
+
+describe "routing redirections", type: :request do
```

Comment:
It turns out that `type: :routing` can't handle the `host!` option or the `redirect_to` assertion. It probably works at a lower level, but we actually need to test it full stack.